### PR TITLE
Removed all occurences of geneva

### PR DESCRIPTION
**Pull request by emielsteerneman**
Removed any and all occurences of the geneva drive. All targets are still compiling succesfully.

### CODE REVIEW

File: src/packing.cpp [hunk 1]
```diff
@@ -127,7 +125,7 @@ std::shared_ptr<packed_protocol_message> createRobotPacket(LowLevelRobotCommand
 
     byteArr[7] = static_cast<uint8_t>(  // K C F De UC 3 bits of geneva
         (0b10000000 & (llrc.do_kick << 7)) | (0b01000000 & (llrc.do_chip << 6)) | (0b00100000 & (llrc.kick_chip_forced << 5)) | (0b00010000 & (llrc.debug_info << 4)) |
-        (0b00001000 & (llrc.use_cam_info << 3)) | (0b00000111 & (llrc.geneva_drive_state)));
+        (0b00001000 & (llrc.use_cam_info << 3)) | (0b00000111 & (0))); // TODO : remove geneva state
```
Comment (rolfvdhulst) [inline src/packing.cpp:128]: Can't you fix this already?

**Review by rolfvdhulst — APPROVED** [review]
LGTM. Make sure to test with the robots if this doesn't break the commands.

File: src/packing.cpp (hunk 1 above)
Comment (emielsteerneman) [reply src/packing.cpp:128]: What do you mean?

Comment (rolfvdhulst) [reply src/packing.cpp:128]: The todo; don't you guys plan on removing it from the llrc?

**Pull request merged**